dockerspawner/dockerspawner.py

```diff
@@ -69,6 +80,22 @@ def client(self):
         )
     )
 
+    tls = Bool(False, config=True, help="If True, connect to docker with --tls")
+    tls_verify = Bool(False, config=True, help="If True, connect to docker with --tlsverify")
+    tls_ca = Unicode("", config=True, help="Path to CA certificate for docker TLS")
+    tls_cert = Unicode("", config=True, help="Path to client certificate for docker TLS")
+    tls_key = Unicode("", config=True, help="Path to client key for docker TLS")
```

Comment:
Is there a way to specify the args that are bundled/required?

I'm currently seeing these option groupings:

* `tls=True` - standalone, causes `tls_config` to just be set to `True`.
* If `tls_cert` is set, `tls_key` needs to be as well.
* Any one of `tls_verify`, `tls_ca`, or the combination of `tls_cert` + `tls_key` need to be set for TLS.